contracts/stake-cw20-external-rewards/src/contract.rs

```diff
@@ -596,7 +591,7 @@ mod tests {
         let msg = stake_cw20::msg::ExecuteMsg::AddHook {
             addr: reward_addr.clone(),
         };
-        app.execute_contract(Addr::unchecked(OWNER), staking_contract, &msg, &[]);
+        let _result = app.execute_contract(Addr::unchecked(OWNER), staking_contract, &msg, &[]);
```

Comment:
Probably want to unwrap this. Currently this will fail silently.

Reply:
Done